### PR TITLE
add mordhau

### PR DESCRIPTION
**Pull request by chasx003**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [x] Does your server use a custom docker image?
    * [x] Have you tried to use a generic image?
    * [ ] Did you PR the nesessary changes to make it work?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

Currently using this docker image https://github.com/chasx003/images/tree/Mordhau    
Not sure how to PR an entire branch

### CODE REVIEW

**Comment by JLBP** [comment]
Just tried this egg out, it installs perfectly, but it does not seem to escape the install phase and is left hanging on installing.

**Comment by chasx003** [comment]
Any other details/log? If you re start the server does it work? (The server has to start once for some of the required files to download, and then should work normally)
Another user besides me has told me it worked for them, so I’m sure we can get it sorted out!

**Comment by JLBP** [comment]
You are completely right. My bad. Lack of sleep and fuelled my coffee, It was a complete oversight. Something I keep forgetting when adding a new egg. Please ignore my last message :)

**Comment by parkervcp** [comment]
Install logs are located here - `/srv/daemon/config/servers/<server-uuid>/install.log`

**Comment by chasx003** [comment]
This is what my panel looks like

```
[2019.05.17-00.35.41:229][ 0]LogMordhauGameSession: Verbose: Updating server session
[2019.05.17-00.35.41:266][ 0]LogGameState: Match State Changed from EnteringMap to WaitingToStart
[2019.05.17-00.35.41:267][ 0]LogLoad: Took 13.266540 seconds to LoadMap(/Game/Mordhau/Maps/Grad/FL_Grad)
[2019.05.17-00.35.41:267][ 0]LogLoad: (Engine Initialization) Total time: 15.84 seconds
[2019.05.17-00.35.41:283][ 0]LogNet: UNetDriver::TickDispatch: Very long time between ticks. DeltaTime: 0.02, Realtime: 8.38. IpNetDriver_0
[2019.05.17-00.35.41:285][ 0]LogModuleManager: Loading module PerfCounters
[2019.05.17-00.35.41:312][ 1]LogMordhauGameSession: Verbose: Updating server session
[2019.05.17-00.35.48:321][ 2]LogMordhauGameSession: Session GameSession successfully created
[Pterodactyl Daemon] Server marked as ON
```

I'm not positive, but I think what ticks it over from installing to online is the "Start Configuration" part of the egg, which in this case I've set it as "Session GameSession successfully created".  I picked that string because  it looked to me like it was always there when a server was starting, and was the last line before "keep alive" type logs started being generated.  If you wanted to test it, you could change that part of the egg to something like "Match State Changed from EnteringMap to WaitingToStart" and see.

Did you manually stop the script?  the `FUnixPlatformMisc::RequestExitWithStatus` part of your log, I only see that when I restart or stop my server, and it would be on its own line

**Comment by JLBP** [comment]
I have not manually stopped the script, just left it to run. This is from another fresh install with the same results as the others. 
The only thing on the install.log is the following;
```Success! App '629800' fully installed.
CWorkThreadPool::~CWorkThreadPool: work processing queue not empty: 1 items discarded.
'linux32/steamclient.so' -> '../.steam/sdk32/steamclient.so'```
Ill attach the full log
 just in case you want to go over it

[install.log](https://github.com/parkervcp/eggs/files/3189932/install.log)

**Comment by parkervcp** [comment]
I can't see any reason that would fail to install.

It looks clean to me at least.

**Comment by JLBP** [comment]
It does install fine and the server does boot up, I just can't understand why it doesn't flick over from installing on the panel.

File: source_servers/mordhau/README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ MORDHAU is a medieval first & third person multiplayer slasher. Enter a hectic b
 
 
 ### Server Ports
-Mordhau requires 2 ports
+Mordhau requires 3 ports
 
 game port (default 7777)
```
Comment (parkervcp) [inline source_servers/mordhau/README.md]: all I can find says it needs 7777 and 7778

Comment (parkervcp) [reply source_servers/mordhau/README.md]: Also needs 4380 for some reason.

Comment (chasx003) [reply source_servers/mordhau/README.md]: I'm not using 4380, so not sure what it is for.  

The beacon port was added in this patch https://steamcommunity.com/games/629760/announcements/detail/1617266961416483617

**Comment by SebastianZuchtBude** [comment]
⁣BlueMail for Android herunterladen ​

**Pull request merged**